notebooks/document-chunking/with-langchain-splitters.ipynb

```diff
@@ -550,21 +549,19 @@
    "source": [
     "from langchain_elasticsearch import (\n",
     "    ElasticsearchStore,\n",
-    "    ApproxRetrievalStrategy,\n",
+    "    DenseVectorStrategy,\n",
     ")\n",
     "from typing import List, Union\n",
     "from langchain_core.documents import Document\n",
     "\n",
     "\n",
-    "class CustomRetrievalStrategy(ApproxRetrievalStrategy):\n",
-    "\n",
-    "    def query(\n",
+    "class CustomRetrievalStrategy(DenseVectorStrategy):\n",
+    "    def es_query(\n",
```

Comment:
The name of this method is the same as a local variable inside it. The code will run, but it would be best to not duplicate names, to avoid confusing people.